doc/sphinx/source/frameworks/tensorflow_connect.rst

```diff
@@ -1,14 +1,60 @@
 .. frameworks/tensorflow_connect.rst:
 
-Connect TensorFlow\*
-====================
+Getting Started with TensorFlow\*
+=================================
 
+No matter what your level of experience with :abbr:`Deep Learning (DL)` systems 
+may be, nGraph provides a path to start working with the DL stack. Let's begin 
+with the easiest and most straightforward options.
 
-See the `README`_ on the `ngraph_bridge repo`_ for the many ways to connect 
-Tensorflow to nGraph, enabling a `DSO`_ backend that can speed up your TensorFlow 
-training and inference workloads.
+Using ``pip`` install
+----------------------
 
+The easiest way to get started is to use the latest PyPI `ngraph-tensorflow-bridge`_,
+which has instructions for Linux* systems, and tips for users of Mac OS X. 
 
+You can install TensorFlow and nGraph to a virtual environment; otherwise, the code 
+will install to a system location.
+
+.. code-block:: console
+   
+   pip install tensorflow
```

Comment:
Please mention the version of tf to be installed via pip (like in the bridge's readme). We might run into issues due to version mismatch between TF and ng-tf-bridge
https://github.com/tensorflow/ngraph-bridge

Note for ng-tf-bridge team, update this doc when upgrading the tf version